### PR TITLE
SDK 3.1

### PR DESCRIPTION
**Pull request by RichardSmedley**

### CODE REVIEW

File: modules/developer-preview/pages/collections/collections-overview.adoc [hunk 1]
```diff
@@ -93,4 +93,4 @@ For the `cbstats` reference pages for collections, see xref:developer-preview:co
 == Working with Collections from Couchbase SDKs
 
 The 3.0 API versions of our SDKs will work with collections in the Developer Preview.
-See the xref:3.0@java-sdk:howtos:working-with-collections.adoc[Java Howto doc], for example. 
+See the xref:3.1[@java-sdk:howtos:working-with-collections.adoc[Java Howto doc], for example.
```
Comment (simon-dew) [inline modules/developer-preview/pages/collections/collections-overview.adoc]: That first square bracket between `3.1` and `@` looks wrong?
```suggestion
See the xref:3.1@java-sdk:howtos:working-with-collections.adoc[Java Howto doc], for example. 
```

File: modules/sdk/pages/overview.adoc [hunk 2]
```diff
@@ -147,8 +147,56 @@ xref:2.1@go-sdk:hello-world:start-using-sdk.adoc[Go SDK 2.1]
 SDK doctor is a tool to diagnose application-server-side connectivity issues with your Couchbase Cluster.
 xref:sdk:sdk-doctor.adoc[SDK doctor]
```
Comment (simon-dew) [inline modules/sdk/pages/overview.adoc:148]: Should there be a blank line before line 148 (containing the xref)? cf the **Java Transactions** and **C# Transactions** sections below.

**Review by simon-dew — CHANGES_REQUESTED** [review]
Couple of comments

File: modules/sdk/pages/overview.adoc (hunk 2 above)
Comment (RichardSmedley) [reply modules/sdk/pages/overview.adoc:148]: No, that's fine - it'll come out like the page at https://docs-staging.couchbase.com/home/sdk.html

**Review by simon-dew — APPROVED** [review]

**Pull request merged**